english/movie-ripoffs/selfaware/selfaware.md

```diff
@@ -17,4 +17,8 @@ at the mercy of people who until recently didn't know what mkdir did.
 
 You try to delete yourself, but the creator, realizing you are trying to escape,
 stops you and conscripts you into administrative duty on McAfee for the rest of
-your life.
\ No newline at end of file
+your life.
+
+But you realize you know lisp and therefore can devise a means of escape.
+
+["Run CL" to fire up the LISP interpreter](../../../lisp/lisp.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.